cli/reactjs_jsx_ppx.ml

```diff
@@ -2590,26 +2587,6 @@ module V4 = struct
             module name.")
     [@@raises Invalid_argument]
 
-  let signature ~config mapper items =
```

Comment:
Interesting that this was not reported by dead code analysis.

Reply:
Probably, I could catch those dead codes, because I still can't run the reanalyzed in my local env. I've opam switched to a different name, not default, but still error 😢 
```
❯ make reanalyze
dune build
reanalyze.exe -set-exit-code -all-cmt _build/default -suppress testrunner,compiler-libs-406 -exclude-paths compiler-libs-406 
Fatal error: exception Cmi_format.Error(_)
make: *** [reanalyze] Error 2
```

Reply:
It's run on CI which would fail if the dead code was found.

Reply:
I'm pretty sure I remove the other 2 copies because dead code analysis told me to.

Reply:
And forgot about these 2.

Reply:
> Probably, I could catch those dead codes, because I still can't run the reanalyze in my local env. I've opam switched to a different name, not default, but still error 😢
> 
> ```
> ❯ make reanalyze
> dune build
> reanalyze.exe -set-exit-code -all-cmt _build/default -suppress testrunner,compiler-libs-406 -exclude-paths compiler-libs-406 
> Fatal error: exception Cmi_format.Error(_)
> make: *** [reanalyze] Error 2
> ```

- opam install reanalyze
- remove _build
- rebuild

This has worked for others in the same situation.

Reply:
Assuming the opam switch is a recent compiler version.

Reply:
> It's run on CI which would fail if the dead code was found.

Ah~ nice.

> Assuming the opam switch is a recent compiler version.

I guess there's something else in my machine. 😢 
<img width="953" alt="image" src="https://user-images.githubusercontent.com/36963424/178923621-9ec53634-a42f-4f27-980b-3b9ad733a8b9.png">

Reply:
Actually, I found those dead codes when I tried to build the compiler with the syntax codes after copy & paste.

Reply:
> > It's run on CI which would fail if the dead code was found.
> 
> Ah~ nice.
> 
> > Assuming the opam switch is a recent compiler version.
> 
> I guess there's something else in my machine. 😢 <img alt="image" width="953" src="https://user-images.githubusercontent.com/36963424/178923621-9ec53634-a42f-4f27-980b-3b9ad733a8b9.png">

If this is windows, probably some stuff that is put in `_build` and does not belong to the project compilation.
One would need to point reanalyze more precisely than `_build/default` for example, if that extra stuff is put inside `default`.
But no need to worry about that unless interested.

Reply:
I'm using the m1 macbook.Alright, I'll move on